src/main/scala/de/htwg/se/sudoku/aview/HttpServer.scala

```diff
@@ -0,0 +1,73 @@
+package de.htwg.se.sudoku.aview
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.{ContentTypes, HttpEntity, StatusCodes}
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.{Route, StandardRoute}
+import akka.stream.ActorMaterializer
+import de.htwg.se.sudoku.controller.controllerComponent.ControllerInterface
+
+import scala.concurrent.{ExecutionContextExecutor, Future}
+
+class HttpServer(controller: ControllerInterface) {
+
+  implicit val system: ActorSystem = ActorSystem("system")
+  implicit val materializer: ActorMaterializer = ActorMaterializer()
+  // needed for the future flatMap/onComplete in the end
+  implicit val executionContext: ExecutionContextExecutor = system.dispatcher
+
+  val route: Route = get {
+    pathSingleSlash {
+      complete(HttpEntity(ContentTypes.`text/plain(UTF-8)`, "<h1>HTWG Sudoku</h1>"))
+    }
+    pathPrefix("sudoku" / "rest" / "v1") {
```

Comment:
* shouldn't this be `rest/v1/sudoku/...` or just `rest/v1/,,,`?
* other students might be confused why there is a `/rest/v1/` in the route

Reply:
1. Don't know what Marko had in mind while implementing the [class](https://github.com/markoboger/de.htwg.se.SudokuInScala/blob/SA08_MicroService/src/main/scala/de/htwg/se/sudoku/aview/HttpServer.scala#L25) which uses this prefix...maybe we should ask him but I didn't introduce it and don't know what his purpose was to do that...
1. Hmm the task was is to create an REST API so they shouldn't be that confused about the `rest` prefix. Also to versioning isn't that fancy and is very appreciated if they think about this kind of things (in my opinion) xD